src/spectrecoff-cli/commands/Table.fs

```diff
@@ -14,43 +14,41 @@ type TableExample() =
     override _.Execute(_context, _) =
 
         let headers = [
-            DefaultHeader (Simple "Firstname")
-            CustomHeader ((Simple "Age"), { defaultColumnLayout with Alignment = Right })
+            DefaultHeader (Calm "Number")
+            DefaultHeader (Calm "Square")
+            CustomHeader ((Calm "Cube"), { defaultColumnLayout with Alignment = Right })
         ]
 
         let rows = [
-            Numbers [11; 37]
-            Strings ["Frank"; "Farmington"]
+            for i in 1 .. 10 -> Numbers [i; pown i 2; pown i 3]
         ]
 
         P "This shows a table with a default and custom laid-out column." |> toConsole
```

Comment:
What do you think about composing these with `Many`, somewhat like this:
``` 
Many [
            P "This shows a table with a default and custom laid-out column."
            exampleTable.toOutputPayload
            NL
            P "Rows can be added to the same table later on."
        ] |> toConsole

        Numbers [20; pown 20 2; pown 20 3] |> addRow exampleTable
        exampleTable
        |> toOutputPayload
        |> toConsole

        Many [
            exampleTable.toOutputPayload
            NL
            P "Tables can be nested, contain other Payloads, and be customized"
        ] |> toConsole
```

Reply:
I didn't do that intentionally, as I want the examples of a specific module to focus on that one, and not bring too many other features into the mix, if that makes sense.